frog/imports/client/Wiki/components/Modal/withModalController.js

```diff
@@ -0,0 +1,52 @@
+//@flow
+
+import * as React from 'react';
+
+import {
+  type ShowModalFunctionT,
+  type HideModalFunctionT,
+  type ModalParentPropsT
+} from './types';
+
+/**
```

Comment:
I prefer single line // comments

Reply:
Javadoc style comments can be picked up by IDEs and make it easier to understand the code by hovering over variables and function calls. Happy to revert to `//` however if you want.

Reply:
I think we should use Flow for that instead?